specs/required-inputs/README.md

```diff
@@ -43,17 +47,36 @@ This will be implemented using a new `nimble-icon-asterisk` icon ([named consist
     <label part="label" class="label">
         <slot></slot>
     </label>
-    ${when(x => x.required, html`
+    ${when(x => x.requiredVisible, html`
         <nimble-icon-asterisk severity="error"></nimble-icon-asterisk>
     `)}
 </div>
 ```
 
-Note that for text-field, text-area, and number-field, we will also remove the visual if the control has the `readonly` attribute set, since [validation is not enforced on a readonly input](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#attribute_interactions). This doesn't apply to select, combobox, or radio buttons, because they do not support `readonly`.
+Note that for text-field, text-area, and number-field, the `readonly` attribute will prevent the display of the asterisk icon, since [validation is not enforced on a readonly input](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#attribute_interactions). This doesn't apply to select, combobox, or radio buttons, because they do not support `readonly`.
```

Comment:
1. It's not clear to me that the required visual should be coupled to the readonly state. Like if the controls are currently marked `disabled` or `readonly` because the user does not have permission to edit them I don't know why the `required-visible` visual would be forced off. 
    
    It could be useful for the user to know what fields are required fields even if they don't have permissions, i.e. they need to ping an admin with permission the details to fill out the form. I think the `required-visible` visual is information on how to fill out the form, not the current state of the form.
2. The mdn docs make the clarification that readonly is not supported by non-text controls because they are semantically equivalent to disabled controls. But I dunno if either of those should impact the `required-visible` visual.

I think this is more an interaction design question. Should the `required-visible` visual always be shown so users understand the details / requirements of the form, not the current state of the form (which is already communicated through different styles for readonly / disabled controls).

Reply:
If an input is required, empty, and readonly/disabled, then validation isn't enforced, and it might be confusing to be able to submit a form where a "required" input is missing. That said, I think that's enough of a corner case that we can probably just live with that in exchange for the simplicity of not having unexpected interactions we have to document. I'll make the change.